templates/repo/view_list.tmpl

```diff
@@ -29,7 +29,7 @@
 				</a>
 				{{template "repo/commit_status" .LatestCommitStatus}}
 				{{ $commitLink:= printf "%s/commit/%s" .RepoLink .LatestCommit.ID }}
-				<span class="grey has-emoji commit-summary" title="{{.LatestCommit.Summary}}">{{RenderCommitMessageLinkSubject .LatestCommit.Message $.RepoLink $commitLink $.Repository.ComposeMetas}}
+				<span class="grey has-emoji commit-summary" title="{{.LatestCommit.Summary}}"><span class="message-wrapper" style="width: min(calc((max(100vw, 768px) - 768px) * 800 + 20rem), calc((max(100vw, 992px) - 992px) * 800 + 360px), calc((max(100vw, 1200px) - 1200px) * 800 + 500px), calc((min(100vw, 1200px) - 1200px) * (0 - 800) + 700px));">{{RenderCommitMessageLinkSubject .LatestCommit.Message $.RepoLink $commitLink $.Repository.ComposeMetas}}</span>
```

Comment:
@silverwind do you have any ideas how to turn this hideous thing into something we can put in the less and/or make less fragile?

Reply:
Are these the breakpoints that fomantic uses? I guess they should defined using actual `@media` queries. I wonder if fomanic exposes Less variables for the breakpoints which we then could re-use. Or just define the widths ourself at the top of base.less.

Reply:
Yeah these are the breakpoints fomantic uses.

Do you mean use:

```css
@media only screen and (max-width: 767.98px) {
```

etc.

I thought those didn't use the viewport?

Reply:
OK looks like I must have done something stupid there.

Reply:
I meant `@media only screen and (max-width: @breakpoint-1) {`, so we'd define the breakpoint vars ourselves.